syntax = "proto3";

import "gog_entry.proto";
import "igdbapi.proto";
import "steam_entry.proto";

package espy;

message Library {
  repeated GameEntry entry = 1;
  repeated SteamEntry unreconciled_steam_game = 2;
  repeated GogEntry unreconciled_gog_game = 3;
}

message GameEntry {
  igdb.Game game = 1;
  GameDetails details = 2;
  SteamEntry steam_entry = 3;
  GogEntry gog_entry = 4;
}

message GameDetails {
  repeated string tag = 1;
}
